packages/apify/src/enqueue_links/click_elements.ts

```diff
@@ -201,13 +222,8 @@ export async function clickElementsAndInterceptNavigationRequests(options): Prom
     return serializedRequests.map((r) => JSON.parse(r));
 }
 
-/**
- * @param {Page} page
- * @param {Set<*>} requests
- * @return {Function}
- * @ignore
- */
-function createInterceptRequestHandler(page, requests) {
+function createInterceptRequestHandler(page: Page, requests: Set<string>): (req: PuppeteerRequest) => Promise<unknown> | undefined {
+    // @ts-ignore // TODO: Not all code paths return a value.
```

Comment:
Please don't use a ts-ignore directive for that
Your return type shouldn't need the `| undefined` right?